lib/web_ui/lib/src/engine/surface/platform_view.dart

```diff
@@ -12,14 +12,13 @@ class PersistedPlatformView extends PersistedLeafSurface {
   final double width;
   final double height;
 
-  html.HtmlElement _hostElement;
   html.ShadowRoot _shadowRoot;
 
   PersistedPlatformView(this.viewId, this.dx, this.dy, this.width, this.height);
 
   @override
   html.Element createElement() {
-    _hostElement = defaultCreateElement('flt-platform-view');
+    html.Element element = defaultCreateElement('flt-platform-view');
```

Comment:
OK, so the latest version will set the size of both the shadow root element and the PlatformView root element. In addition the overflow property is set to auto on the PlatformView root element. As this property was not set currently, the default behaviour was to let the PlatformView overflow. This will haben even if we set the size of the shadow root (e.g. if a video element is wrapped in a div element).

By setting the overflow property on the shadow root, a user may still chose a different overflow behaviour for the shadow root element. Hope that's ok this way.

Reply:
Let's set the overflow to `hidden`, and also set it on the `element` in the `createElement` method so we don't reset it on every `apply`.